static/src/javascripts-legacy/projects/common/modules/experiments/tests/email-demand-tests.js

```diff
@@ -0,0 +1,163 @@
+define([
+    'lib/$',
+    'bean',
+    'bonzo',
+    'fastdom',
+    'lib/config',
+    'lodash/utilities/template',
+    'lib/robust',
+    'common/modules/email/run-checks',
+    'lib/page',
+    'lib/storage',
+    'common/modules/check-mediator',
+    'raw-loader!common/views/experiments/email-demand-test.html',
+    'ophan/ng'
+], function (
+    $,
+    bean,
+    bonzo,
+    fastdom,
+    config,
+    template,
+    robust,
+    emailRunChecks,
+    page,
+    storage,
+    checkMediator,
+    emailDemandTemplate,
+    ophan
+) {
+    var minimumTrailingParagraphs = 5;
+
+    return function () {
+        this.completeFunction = null;
+        this.id = 'EmailDemandTests';
+        this.start = '2017-03-27';
+        this.expiry = '2017-04-21';
+        this.author = 'Leigh-Anne Mathieson';
+        this.description = 'Show an email sign up that simply tracks clicks and links to a google form for food, business, and cities articles.';
+        this.audience = 1;
+        this.audienceOffset = 0;
+        this.successMeasure = 'Number of clicks';
+        this.audienceCriteria = 'All users';
+        this.dataLinkNames = '';
+        this.idealOutcome = 'People will demonstrate their interest in food, business and/or cities emails';
+
+        var articleBody = '.js-article__body';
+
+        var listConfigs = {
+            food: {
+                listName: 'food',
+                displayName: {
+                    normalText: 'food',
+                    accentedText: 'weekly'
+                },
+                tone: 'feature',
+                headline: "What's for dinner?",
+                description: "Sign-up for our new weekly food email and you’ll get recipes, restaurant reviews and the best" +
+                " of all things culinary. Whether you’re a full-on foodie or a budding gastronaut, we’ve something to sate your appetite",
+                linkOnClick: "https://docs.google.com/forms/d/e/1FAIpQLSeGAacgwIRrUFxKPRSUG-imlqEwUKgVYFhOnJP4__avevZEHw/viewform?usp=sf_link",
+                canRun: function () {
+                    var tags = config.page.keywordIds.concat(config.page.nonKeywordTagIds);
+
+                    return tags.includes('lifeandstyle/food-and-drink') || tags.includes('tone/recipes');
```

Comment:
Unfortunately `includes()` isn't supported by IE at all, which might skew the test results

try `tags.indexOf('lifeandstyle/food-and-drink') > -1`

unless the polyfill stuff we have now on dotcom provides this function? @guardian/dotcom-platform might be able to answer that